docopt.h

```diff
@@ -40,6 +34,12 @@
     #endif
 #endif
 
+#include "docopt_value.h"
+
+#include <map>
+#include <vector>
+#include <string>
```

Comment:
Why were these moved down? Seems like #includes of the STL should not come after #defines?

Reply:
Right, this is only required for `docopt_value.h`